rkt/stage1hash.go

```diff
@@ -170,18 +171,20 @@ func getLocalDefaultStage1Hash(fn *image.Finder) (*types.Hash, error) {
 	exePath, err := os.Readlink("/proc/self/exe")
 	if err != nil {
 		if firstErr != nil {
-			return nil, fmt.Errorf("error finding stage1 images %q and %q in rkt binary directory: %v and %v", defaultStage1Image, imgBase, firstErr, err)
+			innerErr := errwrap.Wrap(firstErr, err)
+			return nil, errwrap.Wrap(fmt.Errorf("error finding stage1 images %q and %q in rkt binary directory", defaultStage1Image, imgBase), innerErr)
 		} else {
-			return nil, fmt.Errorf("error finding stage1 image %q in rkt binary directory: %v", imgBase, err)
+			return nil, errwrap.Wrap(fmt.Errorf("error finding stage1 image %q in rkt binary directory", imgBase), err)
 		}
 	}
 	fallbackPath := filepath.Join(filepath.Dir(exePath), imgBase)
 	s1img, err := fn.FindImage(fallbackPath, "", apps.AppImagePath)
 	if err != nil {
 		if firstErr != nil {
-			return nil, fmt.Errorf("error finding stage1 images %q and %q: %v and %v", defaultStage1Image, fallbackPath, firstErr, err)
+			innerErr := errwrap.Wrap(firstErr, err)
```

Comment:
That is not exactly right, but probably errwrap has rather an error stack instead of an error tree.